### PR TITLE
Fix/#1412

### PR DESCRIPTION
**Pull request by hexyls**
closes https://github.com/protofire/omen-exchange/issues/1412 

Adds support for gs_multi_send and removes safe-apps-sdk as a dependency.

To test: 

1. Reinstall node modules
2. Run Omen locally
3. Go to https://xdai.gnosis-safe.io/app/#/welcome or https://rinkeby.gnosis-safe.io/app/#/welcome and setup a safe if you haven't yet
4. Go to Apps in the left sidebar
5. Select the Wallet Connect app
6. To make sure the wallet connect app you are using is the correct one, make sure the url has the following at the end: `appUrl=https://ipfs.io/ipfs/QmT3VxxfFtfEcvq8AeaoFAyUGxePRa2zisvnxLTrQXU5Uf`
7. In Omen, log in with wallet connect - screen shot the QR code, select the screenshot file and copy it
8. Back in the Wallet Connect app, paste the QR code screen shot
9. Now you should be logged in. When you make transactions on Omen you should be able to confirm transactions from the wallet connect app

### CODE REVIEW

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
Seems that i'm getting this error when trying to do any action on rinkeby
Screenshot 2021-02-11 at 12 38 35

**Comment by hexyls** [comment]
@Mi-Lan Rinkeby has a few different versions of the wallet connect app listed, can you please double check that you are using https://ipfs.io/ipfs/QmT3VxxfFtfEcvq8AeaoFAyUGxePRa2zisvnxLTrQXU5Uf which is the only one that supports gs_multi_send? If you still get the error try clearing local storage and cookies to see if that helps 👍

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
@hexyls yeah seems that it was problem with wallet connect version now i got the proper one! Just seems that when i wan't to buy something it buys it...but i'm getting infite loading on buying shares...what could that be?
Screenshot 2021-02-13 at 23 09 46

**Comment by hexyls** [comment]
Thanks @Mi-Lan, great that confirming a tx worked!

Can you please help me debug the infinite loading? 

After making a tx can you wait 5ish minutes and see if `Safe transaction hash` is displayed in the console anywhere and if a hash value is printed out? Also if there are any failed calls in the devtools network tab to `https://safe-transaction.rinkeby.gnosis.io/api/v1/transactions`

**Comment by pimato** [comment]
Did a lot of actions on Rinkeby and it seems to work fine everywhere!

**Review by pimato — APPROVED** [review]

**Review by Mi-Lan — COMMENTED** [review]
I dunno whats going on...seems that gnosis safe doesn't like me xD I tried yesterday and today to buy/sell and now i can't get safe to setAllowance so that i can spend eth all transactions are failing...Maybe something gas related dunno...
Screenshot 2021-02-16 at 11 44 19
-Heres some transactions 
- https://rinkeby.etherscan.io/tx/0xc8a4392fa5fe1919e52466677f00a983bcc034c8c07def225578b67c06efe875
- https://rinkeby.etherscan.io/tx/0xa7b0eb8d95e09cec95feddb1a0ce9e9b1f2c686be01a5ed42bcb5eae1412839c
- https://rinkeby.etherscan.io/tx/0xeeb6507a534e62d92329dd9584a4b4291ee91059efe237b334b13db6b58757ba
- they are all amongst the failed ones

**Review by Mi-Lan — COMMENTED** [review]
Also i can't see dai balance...
Screenshot 2021-02-16 at 12 10 02
Screenshot 2021-02-16 at 12 09 58

**Review by KadenZipfel — APPROVED** [review]
Ran into this issue https://github.com/protofire/omen-exchange/issues/1661 and haven't been able to test, but the code lgtm

**Comment by hexyls** [comment]
@Mi-Lan thanks so much for patiently trying this out, appreciate it 🙌

Could you please try the following when you get a sec:
1. Delete node_modules and reinstall
2. Run Omen
3. Clear local storage on both localhost omen and on the gnosis-safe wallet connect app
4. See if you can login and do stuff

Let me know how it goes!

File: app/src/hooks/connectedWeb3.tsx [hunk 1]
```diff
@@ -64,12 +64,17 @@ export const ConnectedWeb3: React.FC = props => {
...
+
     // disabled block tracker
-    const infura = connectors['Infura']
-    // @ts-expect-error ignore
-    if (infura.engine && infura.engine._blockTracker && infura.engine._blockTracker._isRunning) {
-      // @ts-expect-error ignore
-      infura.engine.stop()
+    if (context.connector) {
+      if (
+        context.connector.engine &&
```
Comment (pRoy24) [inline app/src/hooks/connectedWeb3.tsx:72]: Nit: Not sure if we are using lodash but better to use lodash _.get or a utility function to check for existence of nested properties.

**Review by pRoy24 — APPROVED** [review]
Tried a few actions create/buy/sell pool and outcome tokens. Worked for me everywhere.
On a separate note, I got a "Could not find market error" after a market was created but I assume that this is not related.

**Pull request merged**